site/en/blog/optimizing-lcp-using-signed-exchanges/index.md

```diff
@@ -0,0 +1,463 @@
+
+<figure>
+  {% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/tqpW8Jy95f1jYJeV5Opx.png", alt="Google Analytics segment editor with recommended filters", width="800", height="441" %}
+</figure>
+
+Create a copy of this segment, named "SXG", except with `isSXG` exactly matches `true`.
+
+In your site template, add the following snippet above the Google Analytics snippet. This is a [special syntax](https://github.com/google/sxg-rs#sxg-only-behavior) that ASX will change `false` to `true` when generating a SXG:
+
+```html
+<script data-issxg-var>window.isSXG=false</script>
+```
+
+Customize your Google Analytics reporting script [as recommended](https://github.com/GoogleChrome/web-vitals#send-the-results-to-google-analytics) to record LCP. If you're using gtag.js, modify the `'config'` command to set the custom dimension (replacing `'dimension1'` and `'dimension2'` with the names that Google Analytics says to use):
+
+```js
+gtag('config', 'YOUR_TRACKING_ID', {
+  'dimension1': String(isSXG),
+  'dimension2': document.referrer,
+});
+```
+
+If you're using analytics.js, modify the `'create'` command as [documented here](https://developers.google.com/analytics/devguides/collection/analyticsjs/creating-trackers).
+
+After waiting a few days to collect some data, go to the Google Analytics Events report and add a drilldown for the SXG segment. This should fill in the X for "SXGs affect X% of page views":
+
+<figure>
+  {% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/MNaYo8ZjOxto57txJvwf.png", alt="Google Analytics Events report with SXG segment, showing 12.5% Unique Events", width="363", height="111" %}
+</figure>
+
+Finally, go to the [Web Vitals Report](https://web-vitals-report.web.app/), select "Choose segments", and select "SXG counterfactual" and "SXG".
+
+<figure>
+  {% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/tIsSMUofs9lMv3C0ncwe.png", alt="Web Vitals Report with selections for SXG counterfactual and SXG", width="657", height="164" %}
+</figure>
+
+Click "Submit", and you should see LCP distributions for the two segments. This should fill in the Y for "improving their LCP by Y milliseconds at the 75th percentile":
+
+<figure>
+  {% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/LTiuLf05jVOH2xe9RMOg.png", alt="Web Vitals Report showing LCP distributions for SXG counterfactual and SXG", width="800", height="363" %}
+</figure>
+
+#### Caveats
+
+Once you've applied all of the above filters, SXG counterfactual page loads should consist of things like these:
+
+- cache misses—If the Google SXG Cache doesn't have a fresh copy of the SXG for a given URL, it will redirect to the original URL at your site.
+- other result types—Currently, Google Search only supports SXG for standard web results and a few other types. Others, like Featured Snippets and Top Stories Carousel, will link to the original URL at your site.
+- ineligible URLs—If some pages on your site are not eligible for SXG (e.g. because they are not cacheable), they could appear in this set.
```

Comment:
```suggestion
- **Cache misses:** If the Google SXG Cache doesn't have a fresh copy of the SXG for a given URL, it will redirect to the original URL at your site.
- **Other result types:** Currently, Google Search only supports SXG for standard web results and a few other types. Others, like Featured Snippets and Top Stories Carousel, will link to the original URL at your site.
- **Ineligible URLs:** If some pages on your site are not eligible for SXG (e.g. because they are not cacheable), they could appear in this set.
```